src/main/java/org/prebid/server/proto/openrtb/ext/request/gothamads/GothamAdsImpExt.java

```diff
@@ -0,0 +1,12 @@
+package org.prebid.server.proto.openrtb.ext.request.gothamads;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Value;
+
+@Value
```

Comment:
We prefer static constructor methods calls to `new` calls, so please, replace with `@Value(staticConstructor="of")`.

Reply:
Consistency again.

Reply:
it's becoming a magic word, where can I read about it? Is there any definition of it on the confluence?